how-to-guides/present-cross-validation-results/notebooks/Neptune_cross_validation.ipynb

```diff
@@ -0,0 +1,496 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to present CV with Neptune"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "THIzFFk81nme"
+   },
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "When training models with cross-validation, you can use Neptune namespaces to organize, visualize and compare models.\n",
+    "\n",
+    "By the end of this guide, you will learn how to organize your run to track cross-validation metadata, so that you can easily analyze the results.\n",
+    "\n",
+    "[See this example in Neptune](https://app.neptune.ai/o/common/org/showroom/e/SHOW-3700)\n",
+    "\n",
+    "[![image](https://files.gitbook.com/v0/b/gitbook-x-prod.appspot.com/o/spaces%2F-MT0sYKbymfLAAtTq4-t%2Fuploads%2FLRq4l78xEp31FFS5WzFM%2Fimage.png?alt=media&token=be868c83-7b1b-4314-b138-677531720f0e)](https://app.neptune.ai/o/common/org/showroom/e/SHOW-3700)\n",
+    "<center><small>CV results presented in Neptune UI</small></center>\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "Make sure that you have:\n",
+    "* [Python 3.7+ installed](https://www.python.org/downloads/),\n",
+    "* [Basic familiarity with Neptune (create run and log metadata to it)](hhttps://docs.neptune.ai/you-should-know/what-can-you-log-and-display),\n",
+    "* Familiarity with cross-validation techniques in machine learning."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Install dependencies**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "54CFqKRANLK5",
+    "outputId": "a19f2835-22ab-41c9-ab13-e6e4d8fdbea9",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "! pip install neptune-client torch==1.10.2 torchvision==0.11.3 scikit-learn==1.0.2 #add other dependencies"
```

Comment:
```suggestion
    "! pip install neptune-client torch==1.10.2 torchvision==0.11.3 scikit-learn==1.0.2"
```

Reply:
All dependencies should be provided here

Reply:
@kamil-kaczmarek As far as I know, this project has no missing dependencies.

If there are any specific dependencies I'm missing that you identified, please send them through, and I will add them right away.